packages/office-ui-fabric-react/src/components/TextField/TextField.Props.ts

```diff
@@ -128,6 +128,12 @@ export interface ITextFieldProps extends React.HTMLProps<HTMLInputElement | HTML
    * @default false
    */
   validateOnFocusOut?: boolean;
+
+  /**
+   * Should run validation on Load?
```

Comment:
`validationDisabled` looks more like it will never run validation, we are interested to suppress the validation only at component mount. That is the reason it is defaulted to `true` . Will fix the description to be clearer.